README.md

```diff
@@ -175,8 +179,7 @@ $ python train_reader.py \
     --max_epochs 20 \
     --gpus 4 \
     --precision 16 \
-    --accelerator ddp \
-    --answer_normalization_type dpr
+    --strategy ddp
```

Comment:
The `--accelerator` keyword is deprecated in the latest version of PyTorch Lightning.